### PR TITLE
Add HW8

### PR DESCRIPTION
**Pull request by DariaBoris**

### CODE REVIEW

File: hw8/Task_1.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//
+// > Реализуйте решение двумя способами, используя `function declaration` & `arrow function`
+
+const fibonacci: number[] = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987]
+
+fibonacci.forEach((value: number) => {
+    console.log(value)
+});
+
+fibonacci.forEach((value: number) => console.log(value));
```
Comment (HeorhSemianenia) [inline hw8/Task_1.ts:16]: What is the difference between two ways exept new lines in first function?
image

Comment (HeorhSemianenia) [reply hw8/Task_1.ts:16]: You could not implement function declaration in each tasks, but please do it at leas in 3 tasks and delete redundant arrow functions.

File: hw8/Task_8.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// > Реализуйте решение двумя способами, используя `function declaration` & `arrow function`.
+
+const salad = new Map<string, number>( [
+    ['огурцы', 300],
+    ['помидоры', 200],
+    ['соль', 10],
+    ['сметана', 110],
+]);
+
+salad.forEach((value: number, key: string, ) => {
```
Comment (HeorhSemianenia) [inline hw8/Task_8.ts]: redundant comma with space

File: hw8/Task_9.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// Task 9
+// Используя коллекцию **`Set`** создайте массив неповторяющийхся значений:
+//     ```javascript
+//     const numbers = [1, 6, 3, 2, 7, 325, 234, 54, 586, 23, 24, 64, 56, 3,
+//     5, 7, 8, 1, 6, 8, 9, 4, 5, 61, 2, 4, 5, 6, 3, 5]
+// ```
+//     > Реализуйте решение двумя способами, используя `function declaration` & `arrow function`.
+
+const mySet = new Set ([1, 6, 3, 2, 7, 325, 234, 54, 586, 23, 24, 64, 56, 3, 5, 7, 8, 1, 6, 8, 9, 4, 5, 61, 2, 4, 5, 6, 3, 5]);
+console.log([...mySet])
```
Comment (HeorhSemianenia) [inline hw8/Task_9.ts:11]: Please slightly adjust to make it more readable
const numbers = [1, 6, 3, 2, 7, 325, 234, 54, 586, 23, 24, 64, 56, 3, 5, 7, 8, 1, 6, 8, 9, 4, 5, 61, 2, 4, 5, 6, 3, 5]
const mySet = new Set(numbers);
console.log([...mySet])

File: hw8/Task_1.ts (hunk 1 above)
Comment (DariaBoris) [reply hw8/Task_1.ts:16]: I came up with this but have some doubts:

fibonacci.forEach(function (value: number) {
   console.log(value)
});

Please let me know your thoughts on this.

File: hw8/Task_9.ts (hunk 3 above)
Comment (DariaBoris) [reply hw8/Task_9.ts:11]: Done.

File: hw8/Task_8.ts (hunk 2 above)
Comment (DariaBoris) [reply hw8/Task_8.ts]: Fixed.

File: hw8/Task_1.ts (hunk 1 above)
Comment (DariaBoris) [reply hw8/Task_1.ts:16]: As discussed before, this one seems to be correct. So, I added function declarations for Task 1, Task 2 and Task 3.

Comment (HeorhSemianenia) [reply hw8/Task_1.ts:16]: now it looks good, thanks

**Review by HeorhSemianenia — APPROVED** [review]

**Pull request merged**